src/atlas/projector.rs

```diff
@@ -17,103 +17,101 @@ fn aws_projector<'a>(
-                                    tag.key.as_ref().unwrap().as_ref(),
-                                    tag.value.as_ref().unwrap().as_ref(),
-                                    0,
-                                );
-                                graph.add_edge(
-                                    inst.image_id.as_ref().unwrap().as_str(),
-                                    tag.key.as_ref().unwrap().as_ref(),
-                                    0,
-                                );
-                            }
+                    if let Some(tags) = inst.tags.as_ref() {
+                        for tag in tags {
+                            graph.add_edge(
+                                tag.key.as_ref().unwrap().as_ref(),
+                                tag.value.as_ref().unwrap().as_ref(),
+                                0,
+                            );
+                            graph.add_edge(
+                                inst.image_id.as_ref().unwrap().as_str(),
+                                tag.key.as_ref().unwrap().as_ref(),
+                                0,
+                            );
                         }
-                        None => (),
-                    }
-
-                    // add region info
-                    if opts.all {
-                        graph.add_edge(region, inst.image_id.as_ref().unwrap().as_str(), 0);
                     }
                 }
             }
             AmazonCollection::AmazonResources(resource_map) => {
                 for (res_name, rs) in resource_map {
                     if use_aws_resource(res_name.as_str()) {
                         for r in rs {
-                            // add main edges
-                            graph.add_edge(res_name.as_str(), r.resource_id().unwrap(), 0);
-
                             // add region edges
-                            if opts.all {
-                                graph.add_edge(region, r.resource_id().unwrap(), 0);
+                            if use_global(res_name.as_str()) {
+                                graph.add_edge("global", res_name.as_str(), 0);
+                            } else {
+                                graph.add_edge(region, res_name.as_str(), 0);
                             }
-                        }
-                        if opts.all {
-                            graph.add_edge(region, res_name.as_str(), 0);
+
+                            // add main edges
+                            graph.add_edge(res_name.as_str(), r.resource_id().unwrap(), 0);
                         }
                     }
                 }
             }
+            AmazonCollection::AmazonClusters(clusters) => {
+                for cluster in clusters {
+                    graph.add_edge(region, cluster.cluster_arn().unwrap_or_default(), 0);
+                }
+            }
+            // TODO: possibly remove
+            AmazonCollection::AmazonNetworks(_nets) => (),
```

Comment:
Will need to verify against an account using this to see if it pulls useful additional context that `config` doesn't already grab.